SPECS/nginx/nginx.spec

```diff
@@ -18,6 +18,8 @@ Source1:        nginx.service
 Source2:        https://github.com/nginx/njs/archive/refs/tags/%{njs_version}.tar.gz#/%{name}-njs-%{njs_version}.tar.gz
 Source3:        https://github.com/open-telemetry/opentelemetry-cpp-contrib/archive/%{opentelemetry_cpp_contrib_git_commit}.tar.gz#/opentelemetry-cpp-contrib-%{opentelemetry_cpp_contrib_git_commit}.tar.gz
 Patch0:         CVE-2023-44487.patch
+Patch1:         CVE-2024-7347.1.patch
```

Comment:
I'm not sure that our tooling can handle the .1 and .2
Consider combining or dropping the .1 in the first patch and renaming the second patch something else?

Reply:
I have reached out the Karim to see if the CVE scanning tools can survive this.

Reply:
Is there an explicit reason why we are dividing this .patch file into 2 parts? Why can't we have it in a single patch file?

Reply:
@Camelron can you combine them into 1 patch file?

Reply:
Addressed, please see new patchfile.